### ISSUE TITLE
Dominio A y predicado unario M

### ISSUE THREAD

**Issue by Pipelao** [juanreutter/L-gica2020#11]
Hola @juanreutter 

Tengo una duda con respecto a cómo se ven estas dos cosas, tanto el dominio A como el predicado unario M, ya que ambas corresponden a posiciones del tablero, pero no logro imaginarme cómo ambas pueden ser como unarias por decirlo así, porque al ser un tablero lo interpreto como una grilla con posiciones (x,y), ¿me podría dar un ejemplo tanto del dominio A como de un predicado unario M?

Gracias de antemano.

**Comment by juanreutter** [comment]
claro, piensa en estructuras donde el dominio sea un montón de pares (x,y). 
Luego M es un predicado unario, en una estructura en particular M se va a interpretar como un conjunto de elementos del dominio, en tu caso, un conjunto de pares (x,y). 

(notar que no estoy pidiendo fórmulas que validen que el domino hace sentido desde el punto de vista de una grilla, podría ser algo mucho más loco, un laberinto con portales, qué se yo)

**Comment by Pipelao** [comment]
Ahh perfecto, creo que ahora entiendo, entonces por ejemplo cuando define el predicado binario N para una par (a,b), ¿sería algo así como a = (x1,y1) y b = (x2,y2)?

**Comment by juanreutter** [comment]
Claro

**Comment by Pipelao** [comment]
Perfecto, gracias @juanreutter
